python/tests/plugins/test_gen_protobuf.py

```diff
@@ -138,6 +139,15 @@ def test_gen_protobuf_fails_with_multiple_message_templates(self, load_templates
     def test__convert_camel_case_to_snake_case(self, test_string, expected_string):
         self.assertEqual(expected_string, _convert_camel_case_to_snake_case(test_string))
 
+    @params(
+        ("Data A", "data_a.proto"),
+        ("Message with multiple spaces", "messagewithmultiplespaces.proto"),
```

Comment:
I'm not sure how these two tests don't conflict. Bear with me as I dig in a bit.

Reply:
I was thinking of combining the test....but I wasnt sure if that would be ok to do or not.

Reply:
No, let's keep them separate as they demonstrate different test cases.

The disconnect is because of the snake case function which delineates snake case words by capitalization. So, the input of `Message With Multiple Spaces` would yield `message_with_multiple_spaces.proto`.